crates/sui-core/src/runtime.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use sui_config::NodeConfig;
+use tokio::runtime::Runtime;
+
+pub struct SuiRuntimes {
+    // Order in this struct is the order in which runtimes are stopped
+    pub sui_node: Runtime,
+    pub metrics: Runtime,
+    // json_rpc: Runtime,
+}
+
+impl SuiRuntimes {
+    pub fn new(_confg: &NodeConfig) -> Self {
+        let sui_node = tokio::runtime::Builder::new_multi_thread()
+            .thread_name("sui-node-runtime")
+            .enable_all()
+            .build()
+            .unwrap();
+        let metrics = tokio::runtime::Builder::new_multi_thread()
+            .thread_name("metrics-runtime")
+            .worker_threads(2)
```

Comment:
this was at 4 before, any reason to reduce?

Reply:
No strong opinion here. I thought that 2(and even 1) thread is sufficient, more threads just pollute gdb/htop output. I can put back 4 if you think this makes more sense